.gitignore

```diff
@@ -2,3 +2,6 @@ pkg
 Vagrantfile
 package/sle2docker.8
 package/sle2docker.8.html
+*.sw?
+sle2docker-*.gem
+Gemfile.lock
```

Comment:
A couple of things :grin: 

- What is `*.sw` ? If it isn't strictly coupled with this gem, please remove it.
- Don't add the Gemfile.lock here. We should upload a proper one before releasing the next version.